docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -1267,6 +1293,44 @@ rules, the implementation sorts the rules to avoid conflict; for example, the
 ligature substitution rule for f_f_i will be written before the ligature
 substitution rule for f_i, no matter what their order is in the feature file.
 
+<a name="4.e.i"></a>
+#### 4.e.i Feature Variations (variable font feature replacement)
+
+For variable fonts *only*, it is possible to specify [FeatureVariations](https://docs.microsoft.com/en-us/typography/opentype/spec/chapter2#featurevariations-table), which allow the use of different lookups for different parts of the variation space.
+
+<a name="4.e.i.1"></a>
+##### 4.e.i.1 `conditionset`
+
+A named `conditionset` defines [conditions](https://docs.microsoft.com/en-us/typography/opentype/spec/chapter2#conditionset-table) that can trigger a variable font lookup to swap in different lookups when the conditions are matched. A `conditionset` is simply a list of axis tags and min/max values. A conditionset is matched when all conditions are true. Syntax:
```

Comment:
> A conditionset is matched when all conditions are true.

So the `conditionset` statement can only do boolean `AND` but not `OR`. I wonder if this (or another keyword) should be allowed to define not only "boxes" but also "regions", to borrow @justvanrossum's terminology from 
https://github.com/fonttools/fonttools/blob/9959916c64c5ab24de416aabf05c2bdace610014/Lib/fontTools/varLib/featureVars.py#L22-L29

Reply:
Haven't followed this, but if .fea strictly follows the OTF FeatureVariations structure, it will end up fairly limited in it's practical use. 

One of the reasons fontTools.varLib.featureVars became so complex* is that it allows both AND and OR.

*) so complex that Behdad rewrote it almost from scratch, to make it behave and perform decently, after my initial sloppy implementation

Reply:
What are the situations when "OR" cannot be replaced by separate conditionsets?

Reply:
E.g. `(a OR b) AND (c AND d)` can be rewritten as 

- `a AND c AND d`
- `b AND c AND d`

I may be missing something but if you want 'OR' matching, you just define multiple conditionsets, each describing all 'AND' conditions, no?

Reply:
In FontLab 7, when you assign a special tag to the target glyph, it replaces the source glyph under specified conditions:

```
[srcglyph]~cond1[&cond2][&cond3...]
```

if `srcglyph` is absent, the current glyph’s name without a suffix is used.

Each cond is a 2-letter (internal FL) axis tag with bounds, like `wt>500` or `340>wd` or `100<op<600` etc. — and they're concatenated with `&`.

If I need 'OR', I add a second special tag. 

I know there is some redundancy in this, but I’m genuinely curious if there are conditionsets that involve both OR and AND which cannot be expressed as a list of 'AND' conditionsets.

Reply:
I think Just wasn't at the meeting where we all agreed that FEA should be regarded as assembly-equivalent, to be emitted by programs, and he's still trying to make it easy for humans to write. ;-)

Reply:
> I think Just wasn't at the meeting where we all agreed that FEA should be regarded as assembly-equivalent

TIL. Thanks, and agreed that's a good thing. Back to lurk mode!